views/containers.html

```diff
@@ -4,13 +4,19 @@
 <dd>{{container.name}}</dd>
 <dt>Image</dt>
 <dd>{{container.image}}</dd>
-<dt ng-if-start="!container.ports.length">Port(s)</dt>
-<dd ng-if-end><em>none</em></dd>
-<dt ng-repeat-start='port in container.ports'><span ng-if="$index == 0">Port(s)</span></dt>
-<dd ng-repeat-end>{{port.containerPort}}/{{port.protocol}}<span ng-if="port.hostPort"> to host port {{port.hostPort}}</span></dd>
-<dt ng-if-start="!container.env.length">Env var(s)</dt>
-<dd ng-if-end><em>none</em></dd>
-<dt ng-repeat-start='env in container.env'><span ng-if="$index == 0">Env var(s)</span></dt>
-<dd ng-repeat-end>{{env.name}}={{env.value}}</dd>
+<dt>Ports</dt>
+<dd>
+  <div ng-if="!container.ports.length"><em>none</em></div>
+  <div ng-repeat="port in container.ports | orderBy:port">
```

Comment:
was the `orderBy` a copy/paste from `service.html`? If we want to sort (not a bad idea), should probably sort on `containerPort`